dawn/test/unit-test/dawn/CodeGen/samples/copy_stencil.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+##===-----------------------------------------------------------------------------*- Python -*-===##
+##                          _
+##                         | |
+##                       __| | __ ___      ___ ___
+##                      / _` |/ _` \ \ /\ / / '_  |
+##                     | (_| | (_| |\ V  V /| | | |
+##                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+##
+##
+##  This file is distributed under the MIT License (MIT).
+##  See LICENSE.txt for details.
+##
+##===------------------------------------------------------------------------------------------===##
+
+
+import os
+
+import dawn4py
+from dawn4py.serialization import SIR
+from dawn4py.serialization import utils as sir_utils
+from google.protobuf.json_format import MessageToJson
+
+stencil_name = "copy_stencil"
+output_file = f"{stencil_name}.cpp"
+
+if __name__ == "__main__":
+    interval = sir_utils.make_interval(SIR.Interval.Start, SIR.Interval.End, 0, 0)
+
+    # create the out = in[i+1] statement
+    body_ast = sir_utils.make_ast(
+        [
+            sir_utils.make_assignment_stmt(
+                sir_utils.make_field_access_expr("out", [0, 0, 0]),
+                sir_utils.make_field_access_expr("in", [1, 0, 0]),
+                "=",
+            )
+        ]
+    )
```

Comment:
@Stagno this looks like a really long line. If you can, please turn on the black formatter in your editor.

Reply:
Sorry about that, my editor put this comment on the wrong line. However, there are long lines in here.